packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1296,7 +1296,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 flushMode: () => this.flushMode,
                 reSubmit: this.reSubmit.bind(this),
                 rollback: this.rollback.bind(this),
-                setFlushMode: (mode) => this.setFlushMode(mode),
+                setFlushMode: (mode) => {},
```

Comment:
I'm not sure why we need this callback at all - should it be removed from IRuntimeStateHandler? If it's required for testing purposes, I'd make it optional and not provide here.

Reply:
This removal will be done as part of AB#394